src/utils/hooks/useOutsideClick.ts

```diff
@@ -7,10 +7,14 @@ import {
 type Element = HTMLElement | null;
 type Ref = MutableRefObject<Element>;
 type Refs = MutableRefObject<Element>[];
+type OutsideClickOptions = {
+  useCapture?: boolean;
+};
```

Comment:
I suggest renaming `useCapture` to just `capture`. For two reasons:
* it would be aligned with `options.capture` parameter of `addEventListener()`
* `use` prefix has React hook connotation

Reply:
Good point, done.

Reply:
@udidol please, mark as resolved